strings/accent_universal.json

```diff
@@ -15,9 +15,6 @@
 		"humour": "humor",
 		"saviour": "savior",
 		"the earth": "creation",
-		"penis": "cock",
-		"retarded": "a natural born idiot",
```

Comment:
leave the retarded ones in it converts anyone saying retard or retarded into those lines its better

Reply:
Oh shit didnt realize, lol i just removed it cause i thought it wasn't meant to be there